src/engine/source/wdb/src/wdb.cpp

```diff
@@ -0,0 +1,195 @@
+#include <wdb/wdb.hpp>
+
+#include <iostream>
+#include <unistd.h>
+
+#include <logging/logging.hpp>
+
+#include "unixSocketInterface.hpp"
+
+namespace wazuhdb
+{
+
+WazuhDB::~WazuhDB()
+{
+    if (this->m_fd > 0)
```

Comment:
we should change all the if to Value Operator Variable in order to leave it as stated in the code style